src/main/java/org/folio/rs/service/CheckInItemService.java

```diff
@@ -0,0 +1,47 @@
+package org.folio.rs.service;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+import org.apache.commons.lang3.StringUtils;
+import org.folio.rs.client.CirculationClient;
+import org.folio.rs.client.LocationClient;
+import org.folio.rs.domain.dto.CheckInCirculationRequest;
+import org.folio.rs.domain.dto.CheckInItem;
+import org.folio.rs.repository.LocationMappingsRepository;
+import org.joda.time.DateTime;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+
+import java.util.UUID;
+
+import static org.joda.time.DateTimeZone.UTC;
```

Comment:
Joda time is depricated. Can we use Java DateTime  other instead?

Reply:
Joda time is used in circulation module, it is better to use the same to avoid any serialization issues.

Reply:
If I understand correctly, we just need to have string representation of NOW in the ISO8601? If so, why we cannot use
`Instant.now().truncatedTo(ChronoUnit.MILLIS)`?